refs/biblio.json

```diff
@@ -2425,6 +2425,13 @@
     "SMPTE2052-1": {
         "aliasOf": "SMPTE2052-1-2013"
     },
+    "SMPTE296": {
+        "title": "ST 296:2012 - SMPTE Standard - 1280 × 720 Progressive Image 4:2:2 and 4:4:4 Sample Structure — Analog and Digital Representation and Analog Interface",
+        "href": "https://ieeexplore.ieee.org/document/7291722",
+        "publisher": "SMPTE",
```

Comment:
I would be tempted to say `Society of Motion Picture and Television Engineers (SMPTE)`

Reply:
I was, but all the other entries said SMPTE (similarly ISO not International Standards Organization, W3C not The World Wide Web Consortium) so I went for consistency with other entries.